.vscode/settings.json

```diff
@@ -9,4 +9,10 @@
     "[python]": {
         "editor.defaultFormatter": "charliermarsh.ruff",
     },
+    "python.linting.mypyEnabled": true,
+    "python.linting.enabled": true,
+    "python.testing.pytestArgs": [
+        "tests", "src", "docs"
+    ],
```

Comment:
We looked at this line already together and concluded that everything works fine if you wait long enough for the language server to start. Has anything changed since then?